tests/unit/compat/mock.py

```diff
@@ -0,0 +1,122 @@
+# (c) 2014, Toshio Kuratomi <tkuratomi@ansible.com>
+#
+# This file is part of Ansible
```

Comment:
Do we need to copy Ansible code into our repo? Is it possible to use it from the ansible installation instead?

Reply:
I am not sure whether it can be used from ansible installation or not .I have taken it from "https://github.com/ansible-collections/**community.general**/blob/main/tests/unit/compat/mock.py" .

Reply:
Done